.github/workflows/continuous-benchmarking-teardown.yml

```diff
@@ -81,3 +81,47 @@ jobs:
 
       - name: Delete GCloud Compute Engine VM 
         run: gcloud compute instances delete github-actions-runner-gcr --zone=us-west1-a
+
+  teardown-aws-vm:
+    runs-on: ubuntu-latest
+    env:
+      AWS_RUNNER_NAME: stellar-continuous-aws
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Configure AWS credentials using EASE lab account
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_KEY }}
+          aws-region: us-west-1
+
+      - name: Get self-hosted runner ID
+        id: get-runner-id
+        env:
+          GH_TOKEN: ${{ secrets.DEPLOY_SELF_HOSTED_RUNNER_TOKEN }}
+        run: |
+          gh api \
+            -H "Accept: application/vnd.github+json" \
+            -H "X-GitHub-Api-Version: 2022-11-28" \
+            /repos/vhive-serverless/STeLLAR/actions/runners \
+          | echo id=$(jq '.runners[] | select(.name == "${{ env.AWS_RUNNER_NAME }}") | .id') > $GITHUB_OUTPUT
+
+      - name: Remove self-hosted runner
+        env:
+          GH_TOKEN: ${{ secrets.DEPLOY_SELF_HOSTED_RUNNER_TOKEN }}
+        run: |
+          gh api \
+            --method DELETE \
+            -H "Accept: application/vnd.github+json" \
+            -H "X-GitHub-Api-Version: 2022-11-28" \
+            /repos/vhive-serverless/STeLLAR/actions/runners/${{ steps.get-runner-id.outputs.id }}
+
+      - name: Get AWS EC2 instance ID
+        run: |
+          echo id=$(aws ec2 describe-instances --filters "Name=tag:Name,Values=stellar-continuous-aws-temp" --query "Reservations[*].Instances[*].InstanceId" --output text) > $GITHUB_OUTPUT
```

Comment:
The `tag:Name` value here is `stellar-continuous-aws-temp`, but it looks like during VM creation this is set to `env.AWS_RUNNER_NAME` which seems to be `stellar-continuous-aws` in line 12?

Reply:
Thanks for spotting this! I've corrected it